AirCasting/Utils/Strings.swift

```diff
@@ -305,300 +439,501 @@ struct Strings {
+        static let connectionTimeoutDescription: String = NSLocalizedString("Bluetooth connection failed. Please toggle the power on your device and try again.",
+                                                                            comment: "")
     }
-
+    
     enum ConfirmCreatingSessionView {
-        static let alertTitle: String = LocalizedStringKey("Failure").stringKey
-        static let alertMessage: String = LocalizedStringKey("Failed to create session").stringKey
-        static let contentViewTitle: String = LocalizedStringKey("Are you ready?").stringKey
-        static let contentViewText_1: String = LocalizedStringKey("Your ").stringKey
-        static let contentViewText_2: String = LocalizedStringKey(" session ").stringKey
-        static let contentViewText_3: String = LocalizedStringKey(" is ready to start gathering data.\n\n").stringKey
-        static let contentViewText_4: String = LocalizedStringKey("Hang your AirBeam in a secure position, then press the start recording button below.").stringKey
-        static let contentViewText_4Mobile: String = LocalizedStringKey("Move to your starting location, confirm your location is accurate on the map, then press the start recording button below.").stringKey
-        static let startRecording: String = LocalizedStringKey("Start recording").stringKey
-    }
-
+        static let alertTitle: String = NSLocalizedString("Failure",
+                                                          comment: "")
+        static let alertMessage: String = NSLocalizedString("Failed to create session",
+                                                            comment: "")
+        static let contentViewTitle: String = NSLocalizedString("Are you ready?",
+                                                                comment: "")
+        static let contentViewText_1: String = NSLocalizedString("Your ", comment: "It consists of few parts. Please consider them together. Whole text is following: Your session is ready to start gathering data.")
+        static let contentViewText_2: String = NSLocalizedString(" session ",
+                                                                 comment: "")
+        static let contentViewText_3: String = NSLocalizedString(" is ready to start gathering data.",
+                                                                 comment: "")
+        static let contentViewText_4: String = NSLocalizedString("Hang your AirBeam in a secure position, then press the start recording button below.",
+                                                                 comment: "")
+        static let contentViewText_4Mobile: String = NSLocalizedString("Move to your starting location, confirm your location is accurate on the map, then press the start recording button below.",
+                                                                       comment: "")
+        static let startRecording: String = NSLocalizedString("Start recording",
```

Comment:
all multipart strings here should be merged into single ones using techniques described above